client/src/components/Multiselect.tsx

```diff
@@ -569,7 +569,7 @@ export class Multiselect extends React.Component<IMultiselectProps, any> {
       >
         {hasError && (
           <Alert type="error" variant="primary" closeType="none" role="status">
-            <Trans>Error</Trans>
+            <Trans>Make a selection from the list or clear search text</Trans>
```

Comment:
hmm yeah, this isn'y very clear. It's not necessarily that what they typed is wrong though, since the search bar just narrows the list of options in the dropdown - it's just that they didn't make any selections from the option list dropdown and then tried to apply the filter. So if they have some text in the search bar that indicates they likely misunderstood how it works, and didn't realize they need to select some options from the list after searching.

Reply:
do you think it's worth having that granularity reflected in our error messages?

case 1: input has valid search string but user clicks apply without any selection => `Select at least 1 option`
case 2: input has invalid search string and user clicks apply => `Enter a valid search term` or `Clear search field`

Reply:
I don't think there really are any invalid search terms - I guess just something that doesn't match any options, but then the dropdown says that there are no options matching the search terms. So maybe it should just always raise an error and say to make a selection whenever there are no selections and "apply" is clicked, regardless of whether there is anything in the search bar?

Reply:
Maybe for now we can just merge this in as-is and I'll raise this with design. I know they're doing a pass for all the text this week as well, so can see if they've already rethought this.